template/src/providers.test.ts

```diff
@@ -458,13 +458,14 @@ describe('createHoverProvider', () => {
 describe('createDocumentHighlightProvider', () => {
     it('uses precise document highlights', async () => {
         const result = createDocumentHighlightProvider(
-            () => asyncGeneratorFromValues([[{ range: range1 }, { range: range2 }]]),
+            () => asyncGeneratorFromValues([[{ range: range1 }]]),
+            () => asyncGeneratorFromValues([[{ range: range2 }]]),
```

Comment:
Why change this test?

Reply:
There's a new parameter to `createDocumentHighlightProvider()`: the search-based code intel document highlights provider because Squirrel can provide doc highlights.